examples/find_gamma_eff.py

```diff
@@ -0,0 +1,14 @@
+# PR this needs a docstring, which should make it clear why we are doing this
```

Comment:
This comment was not dealt with, actually -- please supply the docstring.  I actually am not sure what this is for.

Reply:
When not being able to run the GUI, this is used for getting on resonance

Reply:
added updated docstring